docs/intro/tutorial07.txt

```diff
@@ -336,14 +336,14 @@ when loading Django templates; it's a search path.
     Just like the static files, we *could* have all our templates together, in
     one big templates directory, and it would work perfectly well. However,
     templates that belong to a particular application should be placed in that
-    application's template directory (e.g. ``polls/templates``) rather than the
-    project's (``templates``). We'll discuss in more detail in the
+    application's template directory (e.g. :file:`polls/templates`) rather than the
+    project's (:file:`templates`). We'll discuss in more detail in the
```

Comment:
I'm not sure of the value of using `:file:` in all these cases? (It seems more verbose for no gain...) 🤔